lib/ddtrace/ext/distributed.rb

```diff
@@ -5,6 +5,8 @@ module DistributedTracing
       # These are cross-language (eg: Python, Go and other implementations should honor these)
       HTTP_HEADER_TRACE_ID = 'x-datadog-trace-id'.freeze
       HTTP_HEADER_PARENT_ID = 'x-datadog-parent-id'.freeze
+      HTTP_HEADER_SAMPLING_PRIORITY = 'x-datadog-sampling-priority'.freeze
+      SAMPLING_PRIORITY_RANGE = 0...2
```

Comment:
Not sure we want to be that restrictive, we already have some use cases with `sampling_priority == 2` so this should at least be `0..2` and I suspect we want to just support any zero or positive number.